tests/integration/conftest.py

```diff
@@ -1,18 +1,32 @@
+import discord.ext.test
 import pytest
 
 from poglink.bot import ConfigurableBot
 
 
 @pytest.fixture()
-def sample_bot(bans_url_1, rates_url_1):
+def sample_bot(bans_url_1, rates_url_1, event_loop):
+    # Define intents required to publish embeds
+    intents = discord.Intents.default()
+    intents.members = True
+
     bot = ConfigurableBot(
         ".",
         {
             "bans_url": bans_url_1,
             "rates_urls": [rates_url_1],
             "data_dir": "tests/data",
         },
+        loop=event_loop,  # Ensure pytest uses same event loop for async functions as bot
+        intents=intents,
     )
+
+    # Configure bot for testing
+    discord.ext.test.configure(bot)
+
+    # Override rates channel id with test channel id
+    bot.config.rates_channel_id = bot.guilds[0].text_channels[0].id
```

Comment:
This configuration step isn't super well documented but I found this pattern in the tests for `dpytest` [here](https://github.com/CraftSpider/dpytest/blob/ac32da01792e0cffaad93655bf7861adc764dba9/tests/conftest.py#L17-L33)